packages/datepicker/src/ux-datepicker.html

```diff
@@ -1,8 +1,20 @@
-<template class="ux-datepicker">
+<template 
+  class="ux-datepicker ${focused || showDialog ? 'ux-datepicker--focused' : ''} ux-datepicker--${variant} ${textboxValue ? 'ux-datepicker--has-value' : ''}"
+  disabled.bind="disabled & booleanAttr"
+  readonly.bind="readonly & booleanAttr"
+  aria-disabled.bind="disabled & booleanAttr"
+  aria-readonly.bind="readonly & booleanAttr"
+  click.trigger="focusInput()">
```

Comment:
does this mean that any click, either on the input, the datepicker container, or the picker dialog will focus the textbox?

Reply:
No it will not open the picker dialog, only focus the element. The the idea here is to mimic the behavior of `ux-input` where any part of the element is clickable to give focus.